documentation/OptionComparison.md

```diff
@@ -0,0 +1,1332 @@
+  <tr>
+   <td><strong>Option</strong>
+   </td>
+   <td><strong>Cray</strong>
+   </td>
+   <td><strong>GNU</strong>
+   </td>
+   <td><strong>IBM</strong>
+   </td>
+   <td><strong>Intel</strong>
+   </td>
+   <td><strong>PGI</strong>
+   </td>
+   <td><strong>Flang</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Disable compile time range checking
+   </td>
+   <td> N/A
+   </td>
+   <td>fno-range-check
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+  </tr>
+  <tr>
+   <td>Disable call site checking
+   </td>
+   <td>dC
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+  </tr>
+  <tr>
+   <td>Warn for bad call checking
+   </td>
+   <td>eb
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+  </tr>
+  <tr>
+   <td>Set default accessibility of module entities to PRIVATE
+   </td>
+   <td>N/A
+   </td>
+   <td>fmodule-private
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+  </tr>
+  <tr>
+   <td>Force FORALL to use temp
+   </td>
+   <td>N/A
+   </td>
+   <td>ftest-forall-temp
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+   <td>N/A
+   </td>
+  </tr>
+</table>
+
+
+
+## <a name="notes"></a>Notes
+
+**<a name="standards"></a>Standards conformance:** 
+
+All conformance options are similar -- they issue warnings if non-standard features are used.  All defaults are to allow extensions without warnings.  The GNU, IBM, and Intel compilers allow multiple standard levels to be specified.
+
+
+
+*   **Cray**: The capital "-eN" option specifies to issue error messages for non-compliance rather than warnings.  
+*   **GNU:** The "std=_level_" option specifies the standard to which the program is expected to conform.   The default value for std is 'gnu', which specifies a superset of the latest Fortran standard that includes all of the extensions supported by GNU Fortran, although warnings will be given for obsolete extensions not recommended for use in new code. The 'legacy' value is equivalent but without the warnings for obsolete extensions. The 'f95', 'f2003', 'f2008', and 'f2018' values specify strict conformance to the respective standards.  Errors are given for all extensions beyond the relevant language standard, and warnings are given for the Fortran 77 features that are permitted but obsolescent in later standards. '-std=f2008ts' allows the Fortran 2008 standard including the additions of the Technical Specification (TS) 29113 on Further Interoperability of Fortran with C and TS 18508 on Additional Parallel Features in Fortran.  Values for "_level_" are f_95, f2003, f2008, f2008ts, f2018, gnu,_ and _legacy._
+
+**<a name="source"></a>Source format:** 
+
+**Fixed or free source: **  Cray, IBM, and Intel default the source format based on the source file suffix as follows:
```

Comment:
The bolding doesn't seem to happen if there is a space before the final `**`.

Reply:
Thanks!  I fixed this.  GitHub handles this differently from Chrome.  Once I knew what to look for, I found many instances of this that needed to be fixed.